src/halmos/sevm.py

```diff
@@ -1031,6 +1031,15 @@ def run(self, ex0: Exec) -> Tuple[List[Exec], Steps]:
                     w = ex.st.pop()
                     ex.st.push(LShR(ex.st.pop(), w)) # bvlshr
 
+                elif o.op[0] == 'SIGNEXTEND':
+                    w = ex.st.pop()
+                    if not is_bv_value(w): raise ValueError(w)
+
+                    w = int(str(w))
+                    if w > 31: continue
```

Comment:
The `continue` statement should not be used here, as there are remaining steps (i.e., increasing the PC counter and pushing the current node to the worklist) after this if block.

Reply:
Fixed.